components/credentials/src/main/kotlin/org/cloudfoundry/credhub/credential/CertificateCredentialValue.kt

```diff
@@ -55,6 +55,12 @@ class CertificateCredentialValue : CredentialValue {
     @JsonIgnore
     var versionCreatedAt: Instant? = null
 
+    @JsonIgnore
+    var durationOverridden: Boolean = false
+
+    @JsonIgnore
+    var durationUsed: Int = 0
+
     constructor() : super() {}
 
     constructor(
```

Comment:
What a nightmare of a constructore this has become over time. I wanna try to refactor this.